frontend/src/metabase/models/containers/NewModelOptions/tests/common.unit.spec.tsx

```diff
@@ -13,6 +16,22 @@ describe("NewModelOptions (OSS)", () => {
   });
 
   describe("has data access", () => {
+    it("should render loading indicator when fetching databases (metabase#44813)", async () => {
+      fetchMock.get(
+        "path:/api/database",
+        delay(2000).then(() => {
+          return [createMockDatabase()];
+        }),
+      );
+
+      setup({ databases: [createMockDatabase()] });
```

Comment:
WDYT about calling `setup` in the first line of this test, and mocking the endpoint after that with `overwriteRoutes: true` to make it more explicit which mock is going to actually take effect?

```suggestion
      setup({ databases: [createMockDatabase()] });
      
      fetchMock.get(
        "path:/api/database",
        delay(2000).then(() => {
          return [createMockDatabase()];
        }),
        { overwriteRoutes: true },
      );
```